oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -189,10 +251,31 @@ public String getAccount() {
     return this.targetPrincipal;
   }
 
+  @Override
+  public String getQuotaProjectId() {
+    return this.quotaProjectId;
+  }
+
+  public List<String> getDelegates() {
```

Comment:
This method doesn't seem used. Can it be removed?

Reply:
This method is used in tests to verify the delegates.

Reply:
In that case please annotate it as `@VisibleForTesting`. Also, make it non-public if possible. (It might not be but worth checking.)

Reply:
Great, thanks for letting me know the annotation! I added it and removed the "public" to limit the visibility.